evennia/server/portal/ttype.py

```diff
@@ -143,13 +143,23 @@ def will_ttype(self, option):
                     "BEIP",  # > 2.00.206 (late 2009) (BeipMu)
                     "POTATO",  # > 2.00 (maybe earlier)
                     "TINYFUGUE",  # > 4.x (maybe earlier)
-                )
+            )
             ):
                 xterm256 = True
 
+            # use name to identify support for xterm truecolor
+            truecolor = False
+            if (clientname.endswith("-TRUECOLOR") or
+                    clientname in (
+                            "AXMUD",
+                            "TINTIN"
+                    )):
+                truecolor = True
+
```

Comment:
The [TTYPE/MTTS](https://tintin.mudhalla.net/protocols/mtts/) protocol supports `TRUECOLOR` as a valid parameter. With this check it seems you are assuming `truecolor=False` and ignoring the `TTYPE/MTTS` value sent by the client completely. 

I think you should honor the handshake and only override if there is no such value in the communication, or if you know that the given client sends a faulty value (or don't support the proper handshake, which is what most of these client-exceptions are for).

Reply:
Gotcha. I was just following the example set in the case above for xterm256. Thanks for the correction.